api/src/main/java/org/openmrs/module/fgh/mpi/MpiActivator.java

```diff
@@ -104,4 +114,32 @@ protected String getMpiLoggerName() {
 		return getClass().getPackage().getName();
 	}
 	
+	private void registerTask() {
+		
+		try {
+			SchedulerService schedulerService = Context.getService(SchedulerService.class);
+			TaskDefinition taskDefinition = new TaskDefinition();
```

Comment:
Starting here to line number 127 move this code to if block

Reply:
Done